pages/community/index.js

```diff
@@ -35,31 +35,39 @@ export default function Community() {
             There’s community for everyone.
           </Box>
           <Box display="flex">
-            <Button variant="secondary">Watch</Button>
+            <Button variant="tertiary" rounded>
+              Watch
+            </Button>
             <Button variant="link">Find your community</Button>
           </Box>
         </Styled.Hero>
-        <Cell
-          maxWidth={DEFAULT_CONTENT_WIDTH}
-          px="base"
-          py={{ _: 'l', lg: 'xl' }}
-        >
-          <CommunityProvider Component={CardList} />
-        </Cell>
+        <Box>
+          <Cell
+            maxWidth={DEFAULT_CONTENT_WIDTH}
+            px="base"
+            py={{ _: 'l', lg: 'xl' }}
+          >
+            <Box as="section" py={{ _: 'l', lg: 'xl' }}>
+              <CommunitiesProvider Component={CommunityList} />
+            </Box>
+          </Cell>
+        </Box>
         <Box
           display="flex"
           alignItems="center"
           flexDirection="column"
           p="l"
           mb="l"
         >
-          <Box as="h2" mb="l">
+          <Box as="h2" mb="s">
             We’ll help you get connected.
           </Box>
           <Box as="p" mb="l">
             There are hundreds of communities at CF. We’ll help find yours.
           </Box>
-          <Button onClick={handleOnClick}>Find your community</Button>
+          <Button onClick={handleOnClick} rounded>
```

Comment:
Just a personal preference, but I like doing `rounded={true}` to make it more explicit. Feel free to ignore this, though 😅